lib/concurrent-ruby/concurrent/utility/monotonic_time.rb

```diff
@@ -10,25 +10,52 @@ def initialize
 
     if defined?(Process::CLOCK_MONOTONIC)
       # @!visibility private
-      def get_time
-        Process.clock_gettime(Process::CLOCK_MONOTONIC)
+      def get_time(unit)
+        Process.clock_gettime(Process::CLOCK_MONOTONIC, unit)
       end
     elsif Concurrent.on_jruby?
       # @!visibility private
-      def get_time
-        java.lang.System.nanoTime() / 1_000_000_000.0
+      self::TIME_UNITS = Hash.new { |_hash, key| raise ArgumentError, "unexpected unit: #{key}" }.compare_by_identity
```

Comment:
Looking at CI I don't think this syntax worked the way it does now on Ruby 1.9. Do you want to look at that and refactor so it works on older Rubies?

Reply:
Yes, definitely plan to, it's just getting hard to have such old rubies working locally. 

I'll fix that tomorrow and ping you back.

Reply:
I don't know if supporting Ruby 1.9 is an intelligent thing to do anymore?

Reply:
I'd say no, but I have no idea what the support policy of the gem is.